### ISSUE TITLE
Baseline training performance

### ISSUE THREAD

**Issue by tao-sun** [IntelLabs/IntelNeuromorphicDNSChallenge#23]
I ran the baseline model training but performance was not so good as reported in the paper. The SI-SNR.txt is like

# Train        Valid
    9.808843    10.826404
   10.640213    10.979951
   10.750848    11.082418
   10.770719    11.054980
   10.746675    11.096102
   10.768912    11.010266
   10.769256    10.998541
   10.792813    11.049504
   10.786952    10.977900
   10.750953    10.791486
   10.745253    10.933626
   10.740764    10.952813
   10.736836    11.025456

Both the dataset and code used were downloaded from the repo and unmodified and data checksum passed.
I also compare my args.txt with the one under original one under /baseline_solution/sdnn_delays/Trained and all the params were same.
I ran training for two times and got similar results.

Anybody know the cause of this issue or encountered the same situation? 

Thanks!

**Comment by BujSet** [comment]
+1: Not sure what the issue is, but I have similar results from running the baseline, achieves 10.68 SI-SNR for training and 11.00 SI-SNR for validation. I also use the unmodified dataset and have matched the args.

**Comment by tao-sun** [comment]
Setting  lr=0.001 can solve the problem.

**Comment by BujSet** [comment]
Thank you @tao-sun! That seems to have allowed the baseline to converge to 12.44 SI-SNR on the validation set, which is close to the 12.50 that is reported on the metricsboard.
